packaging/linux/openra.metainfo.xml.in

```diff
@@ -42,28 +43,15 @@
       <caption>Multiplayer lobby in the Tiberian Dawn mod</caption>
     </screenshot>
   </screenshots>
-  <url type="homepage">http://www.openra.net</url>
+  <url type="homepage">https://www.openra.net</url>
+  <url type="bugtracker">https://github.com/OpenRA/OpenRA/issues</url>
   <update_contact>paul_at_chote.net</update_contact>
-  <content_rating type="oars-1.0">
-    <content_attribute id="violence-cartoon">none</content_attribute>
-    <content_attribute id="violence-fantasy">none</content_attribute>
+  <content_rating type="oars-1.1">
     <content_attribute id="violence-realistic">moderate</content_attribute>
     <content_attribute id="violence-bloodshed">moderate</content_attribute>
```

Comment:
Maybe also `<content_attribute id="violence-worship">moderate</content_attribute>` if we want to be really pedantic?

>  For example, this would include blowing up a church in a war game or referencing the crusades between the Christians and Muslims.

Reply:
I've added it 2b429cec045d3bf68d0f38bb2b59b6ed7aef38a8